spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/config/reload/ConfigurationUpdateStrategy.java

```diff
@@ -24,7 +24,7 @@
  *
  * @author Nicola Ferraro
  */
-public final record ConfigurationUpdateStrategy(String name, Runnable reloadProcedure) {
```

Comment:
I have Intellij ultimate at my new workplace, and it suggested that `final` is redundant here :)